config/config.go

```diff
@@ -35,6 +35,11 @@ var (
 		Password: "",
 	}
 
+	defaultConnectionPool = ConnectionPool{
+		MaxIdleConns:        100,
+		MaxIdleConnsPerHost: 10,
```

Comment:
you should use `MaxIdleConnsPerHost : 2` as default value so that chproxy stays the same by default

Reply:
I've based on this https://github.com/ContentSquare/chproxy/issues/264#issuecomment-1318931700
Ok, will change it back to 2.